lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -109,6 +109,8 @@ var WebMapServiceCatalogItem = function(terria) {
      */
     this.maxRefreshIntervals = 1000;
 
+    this.displayDuration = undefined;
```

Comment:
Can you add doc for this?  It's especially useful because it gets extracted from the source code and used in the data source editor UI.